bin/mdsctl

```diff
@@ -127,6 +152,8 @@ options:
   --configure:{key}+={value}                          : append configuration value by key
   --configure:{key}=                                  : clears configuration value by key
   --preset:[preset-key[,preset-key]]                  : preset configurations
+  --home                                              : returns installation directory
+  --quiet                                             : less verbose
```

Comment:
Controlling verbosity via an environment variable might be nice at some point. I suppose one could also `alias mdsctl=mdsctl --quiet`

Reply:
yeah ... i envision we will want to do something like ssh's verbosity switch (iirc), in that the more v's one provides the more verbose, eg: ... -vvv